carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/R.java

```diff
@@ -163,17 +168,25 @@ public boolean containsKey(String key) {
     public String get(String key) {
         String value = getTestProperties().getProperty(key);
         if (value != null) {
-            LOGGER.warn("Overriden '" + key + "=" + value + "' property will be used for current test!");
+            LOGGER.warn("Overridden '" + key + "=" + value + "' property will be used for current test!");
             return value;
         }
         
         value = CONFIG.resourceFile.equals(resourceFile) ? PlaceholderResolver.resolve(propertiesHolder.get(resourceFile), key)
                 : propertiesHolder.get(resourceFile).getProperty(key);
+
+        String decryptedText = decrypt(value, CRYPTO_PATTERN);
```

Comment:
Logic is pretty broken. Let's make it in sync with naming, i.e. we should call decrypt only if isEncrypted.
so inside get method do:
if (isEncrypted(...)) {
 value = decrypt(value, CRYPTO_PATTERN);
}
and remove your 180-182 if lines...